source/utils.py

```diff
@@ -53,6 +53,10 @@ def split_dots(value):
     return list(map(int, value.split(".")))
 
 
+def compute_list_average(l):
+    return sum(l) / len(l)
+
+
 def get_datetime_filename():
     return datetime.now().strftime("%d-%m__%H:%M:%S")
 
```

Comment:
I think you need to merge main into your branch before we can merge the pull request. this utils.py version is not the most recent one in town. the Guru Halid will tell us what to do.